pymarl/config/envs/struct_c_10.yaml

```diff
@@ -0,0 +1,23 @@
+env: struct
+
+env_args:
+  n_comp: 10
+  discount_reward: 0.95
+  k_comp: 9
+  state_obs: True
+  state_d_rate: True
+  state_alphas: True
+  obs_d_rate: False
+  obs_multiple: False
+  obs_all_d_rate: False
+  obs_alphas: False
```

Comment:
Shall we add here "True"?

Reply:
[depend on this discussion :D ](https://github.com/moratodpg/marl_struct/pull/16#discussion_r999435084)

Reply:
91bb5bc